### DISCUSSION TITLE
SDB GUI 3.6.1

### DISCUSSION THREAD

**Discussion by rifqiharrys** [rifqiharrys/sdb_gui#9] [General]
## Release Note
- Remove `Auto Negative Sign` option and replace it with depth direction assignment on main window
- Add depth direction option on data save window

<hr /><em>This discussion was created from the release <a href='https://github.com/rifqiharrys/sdb_gui/releases/tag/v3.6.1'>SDB GUI 3.6.1</a>.</em>

**Comment by rifqiharrys** [comment]
Hi Tom,
Thank you for your feedback. As I cannot see your comment on github's
discussion, I will reply to this email. I assume your depth data is in
integer, correct?
I made it that depth data has to be in float so it's making the error pop
up if you select other than float data.
There are sample files in ./notebook/data

Cheers

On Sat, 15 Jun 2024 at 17:53, Tom Shand ***@***.***> wrote:

> Thanks for writing this, looks really useful! I'm having a bit of trouble
> getting it to run though - I have an RGB TIF and have digitised some depths
> (though don't have 'real' depths so have just guessed a few) and loaded
> both and selected the header of my depths but i'm getting a 'Please select
> headers correctly' warning. Do I not have enough depths? If you had an
> example or a sample file that could really help. Thanks!
>
> Screenshot_1.jpg (view on web)
> <https://github.com/rifqiharrys/sdb_gui/assets/63450096/a13996bf-473c-4e6e-9a31-85e92cf8a3bd>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rifqiharrys/sdb_gui/discussions/9#discussioncomment-9781188>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFZPLH6GYJQFNSE2326RXA3ZHQMK3AVCNFSM6AAAAABGUMCZZOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOBRGE4DQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Rifqi Muhammad Harrys
Pusat Pemetaan Kelautan dan Lingkungan Pantai (Center of Marine and Coastal
Mapping),
Badan Informasi Geospasial (Geospatial Information Agency of Indonesia)

**Comment by tdshand** [comment]
Hi Rifqi,
Yes that's probably it - I wrote a comment as I was initially struggling to
get it to recognise the depths in the shapefile - but then I generated them
another way and it worked, so hence I deleted the comment.

Great little program though. Thanks for putting it together!

Best,
Tom
